### DISCUSSION TITLE
Preserving object dates while transfering

### DISCUSSION THREAD

**Discussion by whitepiratebaku** [minio/minio#18059] [Q&A] [answered]
Hi, we are moving objects from folder/filesystem to minio server, like:

mc mv --recursive /data/storage/bucket/* minio/bucket/

Is it possible to do same operation but keep the original date/time from objects?

**Comment by klauspost** [comment, marked as answer]
No. S3 does not allow to override modification time through the API.

**Reply by whitepiratebaku** [reply]
I see, imagine we are doing migration from older minio to latest minio and want to preserve the modification date of objects, that is not possible? no workaround? it is needed for retention policies to work properly. 
Note: it is about following migration, https://min.io/docs/minio/linux/operations/install-deploy-manage/migrate-fs-gateway.html . But not mirring but  using mc mv and doing move from filesystem to minio server

**Reply by harshavardhana** [reply]
No workaround
